x-pack/test/functional/apps/rollup_job/rollup_jobs.js

```diff
@@ -4,64 +4,64 @@
+    //make sure all dates have the same concept of "now"
+    const now = new Date();
+    const pastDates = [
+      datemath.parse('now-1d', { forceNow: now }),
+      datemath.parse('now-2d', { forceNow: now }),
+      datemath.parse('now-3d', { forceNow: now }),
+    ];
 
-    after(async () => await esArchiver.unload('logstash_functional'));
-
-    it('create and save a new job', async () => {
-      const jobName = 'Testjob1';
-      const indexPattern = '.kibana*';
-      const indexName = 'rollup_index';
+    it('create new rollup job', async () => {
       const interval = '1000ms';
 
-      await PageObjects.rollup.createNewRollUpJob();
-      await PageObjects.rollup.verifyStepIsActive(1);
-      await PageObjects.rollup.addRoleNameandIndexPattern(jobName, indexPattern);
-      await PageObjects.rollup.verifyIndexPatternAccepted();
-      await PageObjects.rollup.setIndexName(indexName);
-      await PageObjects.rollup.moveToNextStep();
-
-      //now navigate to histogram
-      await PageObjects.rollup.verifyStepIsActive(2);
-      await PageObjects.rollup.setJobInterval(interval);
-      await PageObjects.rollup.moveToNextStep();
+      pastDates.map(async (day) => {
+        await es.index(mockIndices(day, rollupSourceDataPrepend));
+      });
 
-      //Terms (optional)
-      await PageObjects.rollup.verifyStepIsActive(3);
-      await PageObjects.rollup.moveToNextStep();
+      await PageObjects.common.navigateToApp('rollupJob');
+      await PageObjects.rollup.createNewRollUpJob(rollupJobName, rollupSourceIndexPattern, targetIndexName,
+        interval, ' ', true, { time: '*/10 * * * * ?', cron: true });
 
-      //Histogram(optional)
-      await PageObjects.rollup.verifyStepIsActive(4);
-      await PageObjects.rollup.moveToNextStep();
+      const jobList = await PageObjects.rollup.getJobList();
+      expect(jobList.length).to.be(1);
 
-      //Metrics(optional)
-      await PageObjects.rollup.verifyStepIsActive(5);
-      await PageObjects.rollup.moveToNextStep();
+    });
 
-      //saveJob and verify the name in the list
-      await PageObjects.rollup.verifyStepIsActive(6);
-      await PageObjects.rollup.saveJob();
+    after(async () => {
+      //Stop the running rollup job.
+      await es.transport.request({
+        path: `/_rollup/job/${rollupJobName}/_stop?wait_for_completion=true`,
+        method: 'POST',
+      });
+      // Delete the rollup job.
+      await es.transport.request({
+        path: `/_rollup/job/${rollupJobName}`,
+        method: 'DELETE',
+      });
 
-      // verify jobListTitle
-      const jobList = indexBy(await PageObjects.rollup.getJobList(), 'jobName');
-      log.debug(JSON.stringify(jobList));
-      log.debug(Object.keys(jobList));
-      expect(Object.keys(jobList)).to.have.length(1);
+      //Delete all data indices that were created.
+      await es.indices.delete({ index: 'rollup*' });
+      await es.indices.delete({ index: `${rollupSourceIndexPattern}` });
```

Comment:
We don't need the template literal here. This can be:

```js
await es.indices.delete({ index: rollupSourceIndexPattern });
```

Reply:
Right. Of course.